iOS/Flows/Conversation/Emotion Detail/EmotionDetailCoordinator.swift

```diff
@@ -10,10 +10,10 @@ import Foundation
 
 class EmotionDetailCoordinator: PresentableCoordinator<Void> {
 
-    private let emotions: [Emotion]
+    private let expression: Expression
```

Comment:
Adding this here so we can also display the person gradient. I haven't added that yet.